EnvironmentSimulator/Libraries/esminiLib/esminiLib.cpp

```diff
@@ -1038,6 +1038,18 @@ extern "C"
     }
 
     SE_DLL_API int SE_AddObject(const char *object_name, int object_type, int object_category, int object_role, int model_id)
+    {
+        SE_OSCBoundingBox bb = {{0.0, 0.0, 0.0}, {0.0, 0.0, 0.0}};
+        return SE_AddObjectWithBoundingBox(object_name, object_type, object_category, object_role, model_id, bb, 1);
```

Comment:
Suggestion:
```
return SE_AddObjectWithBoundingBox(object_name,
                                   object_type,
                                   object_category,
                                   object_role,
                                   model_id,
                                   bb,
                                   static_cast<int>(EntityScaleMode::BB_TO_MODEL));
```
I know I suggested "1" :), but now I think use of enum adds transparency